### PR TITLE
AB#100 npm script cleanup

### PR DESCRIPTION
**Pull request by michaelschufi**
Scripts cleanup and standardization + cleaning up cypress code.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -50,9 +50,6 @@ steps:
       npm run test
     displayName: "test"
 
-  - script: |
-      npm run lint
-    displayName: "lint"
```
Comment (mailaenderli) [inline azure-pipelines.yml:55]: Why did you remove the linting?

Comment (michaelschufi) [reply azure-pipelines.yml:55]: See above. The `npm run lint` command ran twice.

**Review by StuckiSimon — APPROVED** [review]
Looks good to me 👍 
I've added a short section for new devs in the project highlighting issues we've seen in previous PRs.
